### PR TITLE
Ran a protocol with a static model file generated by fccodegen.

### PR DESCRIPTION
**Pull request by MichaelClerx**
Seems to work

### CODE REVIEW

File: fc/utility/protocol.py [hunk 1]
```diff
@@ -392,35 +448,105 @@ def SetModel(self, model, useNumba=False, useCython=True, exposeNamedParameters=
...
+                temp_dir = tempfile.mkdtemp(dir=self.outputFolder.path)
+            else:
+                temp_dir = tempfile.mkdtemp()
+
+            # Copy model file
+            model_file = os.path.join(temp_dir, 'model.pyx')
+            shutil.copy(model, model_file)
+
+            # Get path to root dir of fc module
+            fcpath = os.path.abspath(os.path.join(fc.MODULE_DIR, '..'))
```
[the remark this answers is not in the document]
Comment (MichaelClerx) [reply fc/utility/protocol.py]: Guess so! Just copied from above to be honest ;-)

File: test/python/TestGeneratedPyxFile.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    Temporary test for `fccodegen` generated `pyx` model.
+
+    """
+
+    def test_run(self):
+        proto = fc.Protocol('test/protocols/test_generated_pyx.txt')
+        proto.SetOutputFolder('Py_TestGeneratedPyx_test_run')
+        proto.SetModel('test/data/weblab_model.pyx')
+        #proto.Run()
```
[the remark this answers is not in the document]
Comment (MichaelClerx) [reply test/python/TestGeneratedPyxFile.py]: Woops! I was uncommenting and re-running one line at a time and stopped too early

[the remark this answers is not in the document]
Comment (MichaelClerx) [reply test/python/TestGeneratedPyxFile.py]: It runs and does stuff but fails!

[the remark this answers is not in the document]
Comment (jonc125) [reply test/python/TestGeneratedPyxFile.py]: Ah, you need to add a vector output for the state variables. I'll send you an example in Slack.

**Comment by MichaelClerx** [comment]
@jonc125 ready for re-reviewing!

**Pull request merged**